GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/UI/AboutWindowController.xib

```diff
@@ -43,7 +43,7 @@
                                         <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
                                     </textFieldCell>
                                     <connections>
-                                        <binding destination="-2" name="value" keyPath="GitVersion" id="E9O-km-WQh"/>
+                                        <binding destination="-2" name="value" keyPath="gitVersion" id="cD3-Y6-vdy"/>
```

Comment:
What do `keyPath` and `id` refer to?

(I am new to this 😄 )

Reply:
Just to add to @HiKumar-MS's comment - Although this is an XML file, the elements are not documented. Apple provides built-in tool called [Interface Builder](https://developer.apple.com/library/archive/documentation/ToolsLanguages/Conceptual/Xcode_Overview/UsingInterfaceBuilder.html) to visually edit the file.